androidlibrary_lib/src/androidTest/java/org/opendatakit/fragment/AlertNProgessMsgFragmentMgerTest.java

```diff
@@ -61,11 +62,10 @@ public void testAlertDialogIsCreated() {
                 systemUnderTest.createAlertDialog(ALERT_DIALOG_TITLE, ALERT_DIALOG_MESSAGE, fragmentManager, fragmentId);
                 fragmentManager.executePendingTransactions();
 
-                Fragment frag = fragmentManager.findFragmentByTag(ALERT_DIALOG_TAG);
-                assertNotNull(frag);
-                assertThat(frag.isAdded(), is(true));
+                MatcherAssert.assertThat(Objects.requireNonNull(fragmentManager.findFragmentByTag(ALERT_DIALOG_TAG)).isAdded(), is(true));
```

Comment:
This is a more minor point, but changing from assertNotNull to using Objects.requireNonNull is changing the output of the test. When asserting, the test is applying a check... combining the check into a longer statement makes it less obvious where the problem is. AssertNotNull will cause a failure and report the exact problem.

I do not think it's an improvement, so I kindly request you restore assertNotNull

Reply:
> This is a more minor point, but changing from assertNotNull to using Objects.requireNonNull is changing the output of the test. When asserting, the test is applying a check... combining the check into a longer statement makes it less obvious where the problem is. AssertNotNull will cause a failure and report the exact problem.
> 
> I do not think it's an improvement, so I kindly request you restore assertNotNull

Hi @wbrunette I have changed it back. Thank you for the insight